mz_config.h is missed from installation - broken build for all consumers

```
/usr/local/include/minizip-ng/mz.h:164:10: fatal error: 'mz_config.h' file not found
  164 | #include "mz_config.h"
      |          ^~~~~~~~~~~~~
1 error generated.
```
```
/usr/local/include/minizip-ng/mz_os.h:48:10: fatal error: 'mz_config.h' file not found
   48 | #include "mz_config.h"
      |          ^~~~~~~~~~~~~
1 error generated.
```

`#include "mz_config.h"` added in this commit: https://github.com/zlib-ng/minizip-ng/commit/2f074dc2b677154fd3fa181f6452ade41c8d36e8

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -169,6 +169,7 @@ check_function_exists(symlink HAVE_SYMLINK)
 check_function_exists(readlink HAVE_READLINK)
 
 configure_file(mz_config.h.cmakein ${CMAKE_CURRENT_SOURCE_DIR}/mz_config.h @ONLY NEWLINE_STYLE UNIX)
+list(APPEND MINIZIP_HDR mz_config.h)
 
 if(MZ_LIBCOMP)
     if(APPLE)

